code/modules/mob/living/silicon/robot/drone/drone_abilities.dm

```diff
@@ -49,35 +49,6 @@
 	else
 		..()
 
-/mob/living/silicon/robot/drone/verb/customize()
```

Comment:
No one in the server has a custom drone skin, and we closed fluff items. This wont be missed.